easybuild/easyconfigs/l/libfabric/libfabric-1.12.1-GCCcore-10.3.0.eb

```diff
@@ -43,6 +43,7 @@ builddependencies = [
 
 dependencies = [
     ('numactl', '2.0.14'),
+    ('PSM2', '12.0.1'),
```

Comment:
As discussed during today's EasyBuild conf call, this should be done conditionally, only for `x86_64` since PSM2 is not compatible with Arm (and probably also not with RISC-V).

@SebastianAchilles will open a PR for that...

Reply:
Done in https://github.com/easybuilders/easybuild-easyconfigs/pull/20585